src/plugins/github/fetchGithubOrgTest.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -eu
+
+data_file=src/plugins/github/example/example-organization.json
+
+usage() {
+  printf 'usage: %s [-u|--updateSnapshot] [--[no-]build] [--help]\n' "$0"
+  printf 'Required environment variables:\n'
+  printf '  GITHUB_TOKEN: A 40-character hex string API token.\n'
+  printf 'Flags:\n'
+  printf '  -u|--updateSnapshot\n'
+  printf '      Update the stored file instead of checking its contents\n'
+  printf ' --[no-]build\n'
+  printf '      Whether to run "yarn backend" before the test.\n'
+  printf '      Default is --build.\n'
+  printf '  --help\n'
+  printf '      Show this message\n'
+  printf '\n'
+  printf 'Environment variables:'
+  printf '  BIN\n'
```

Comment:
Wait, I don’t understand; why isn’t this called `SOURCECRED_BIN`? The
test harness relies on that to properly link the output directory; does
this still work if you `rm -r backend && yarn test --full`?

Reply:
Good catch. I think this was a casualty of my using `s/SOURCECRED_//g` at some point while messing around with the github token variable.